server/routes/router.js

```diff
@@ -4,24 +4,20 @@ var axios = require('axios');
 var users = require("../models/userAuth");
```

Comment:
It's ES6 I think. Const (and maybe let) might be a better solution?

Reply:
I agree `const` is what should be used, I didn't really want to change around things that might affect functionality in some eventuality. There are also some `require`ments that aren't being used (`axios` and `mongoose`). Will make the imports `const`